mlrun/wasbfs/fs.py

```diff
@@ -0,0 +1,104 @@
+import re
+
+from adlfs import AzureBlobFileSystem
+
+
+class WasbFS(AzureBlobFileSystem):
+
+    protocol = "wasb"
+
+    def _convert_wasb_schema_to_az(self, url):
```

Comment:
This seems to be the main function you use in each class method, add docstring for the function.

Reply:
Done